src/main/java/com/cse/ngsa/app/middleware/RequestLogger.java

```diff
@@ -1,10 +1,15 @@
 package com.cse.ngsa.app.middleware;
 
 import java.net.InetSocketAddress;
-import java.text.MessageFormat;
+import java.time.Instant;
+import java.util.Arrays;
+
```

Comment:
Remove the blank lines on the imports.  I think the linter emits a warning for this, but I don't recall exactly.  In any case, kill the blank lines and be sure that there aren't any linter issues.